sdk/azstart/go.mod

```diff
@@ -0,0 +1,24 @@
+module azstart
+
+go 1.19
```

Comment:
For consistency with our SDKs, this should be 1.18

Reply:
downgraded to 1.18